[BLOG] Add Captioning: A Newcomer's Guide (#828)
Quansight/Quansight-website: apps/labs/posts/captioning-a-newcomers-guide.md

---
title: 'Captioning: A Newcomer’s Guide'

> published: February 1, 2024

Reviewer: +/- a day won't matter much. By the time this goes live it'll be Wednesday somewhere on Earth, anyways, so this'll be accurate either day it goes live.

```suggestion
published: January 24, 2024
```

Legally, this may be a requirement for certain organizations.  There are also many specific cases requiring captions you may not expect, like [rules about made-for-TV content on the Internet](https://www.fcc.gov/consumers/guides/captioning-internet-video-programming). Make sure to research thoroughly or consult an expert. 

As mentioned in *Why should I caption things?* the [W3C has also complied a list of business case arguments in favor of accessibility](https://www.w3.org/WAI/business-case/) accommodations such as captions.

### Are there criteria for this? 

As far as I know, there is not a singular standard for captioning content, file format, and styling that are used absolutely everywhere. However, there are only a few dominant ones. Once again, [WCAG covers this as “alternatives for time-based media”](https://www.w3.org/TR/WCAG22/#time-based-media) and navigating to the [Understanding articles](https://www.w3.org/WAI/WCAG22/Understanding/time-based-media.html) per guideline will provide you with examples of how and how not to caption.

A few more tips: 

> * Follow the file format or other formatting relevant to where you are hosting the content. This ensures it behaves as expected. If you are hiring a captioner, know what service you will be hosting onahead of time so both sides can check they have the appropriate format and tech support.

Reviewer 2: ```suggestion
* Follow the file format or other formatting relevant to where you are hosting the content. This ensures it behaves as expected. If you are hiring a captioner, know what service you will be hosting on ahead of time so both sides can check they have the appropriate format and tech support.
```

* Word of mouth was by far the most helpful for finding reliable live captioning. Start asking people you have who run events. If a captioner replies they aren’t available, you may want to ask if they recommend anyone for you to contact instead.
* If you know someone in your community who had a good experience with a captioner, ask if they’d be willing to introduce you or pass on the contact info. The bonus is that these captioners might have more familiarity with the terms used in your community and better know how to caption them. I bring this up because this was really tricky with the many software homophones (and affinity for switching out or removing vowels in tech names) in captioning for events in software and scientific spaces.

> * Make time to do a dry run of your event. Ask the if the captioner is interested in being involved in it and take the time to test out the tech support aspects. If they say they are not interested, accept that.

Reviewer 2: ```suggestion
* Make time to do a dry run of your event. Ask if the captioner is interested in being involved in it and take the time to test out the tech support aspects. If they say they are not interested, accept that.
```

* Plan time ahead of your event to get the captioner set up and running. Test it before you have attendees watching.

> * Ask your captioner they want the event materials ahead of time, and if so how much ahead of time? If they say they do not want materials ahead of time, accept that. This is likely to depend on the captioner’s preferences and their comfort level with the vocabulary in your field, so it’s best to let them decide but still good to offer.

Reviewer 2: ```suggestion
* Ask your captioner if they want the event materials ahead of time, and if so how much ahead of time? If they say they do not want materials ahead of time, accept that. This is likely to depend on the captioner’s preferences and their comfort level with the vocabulary in your field, so it’s best to let them decide but still good to offer.
```
